src/parser/jobs/gnb/modules/SonicBreak.tsx

```diff
@@ -0,0 +1,173 @@
+		return _.last(this.SonicBreakApplications)
+	}
+
+	override initialise() {
+		const playerFilter = filter<Event>().source(this.parser.actor.id)
+		this.addEventHook(playerFilter.type('statusApply').status(this.data.statuses.SONIC_BREAK.id), this.onDotApply)
+		this.addEventHook(playerFilter.type('damage'), this.onDotDamage)
+		this.addEventHook(playerFilter.type('statusRemove').status(this.data.statuses.SONIC_BREAK.id), this.onDotRemove)
+		this.addEventHook('complete', this.onComplete)
+
+	}
+
+	private onDotApply(event: Events['statusApply']) {
+		const lastSonicBreak = this.lastSonicBreakApplication
+
+		//If below is true, we broke it and the last window was not closed properly.
+		if (lastSonicBreak != null && lastSonicBreak.end == null) {
+			lastSonicBreak.end = (lastSonicBreak.start + this.data.statuses.SONIC_BREAK.duration)
+		}
+
+		this.breakMaker(event)
+
+	}
+
+	private onDotDamage(event: Events['damage']) {
+		if (event.cause.type === 'status') {
+			if (event.targets.find(e => e.amount !== 0) && event.cause.status === this.data.statuses.SONIC_BREAK.id) { //Dot did damage, increment.
+				this.breakUpdater(event)
+			}
+		}
+	}
+
+	private onDotRemove(event: Events['statusRemove']) {
+		this.breakEnder(event)
+	}
+
+	private breakMaker(event: Event) : void {
+		const sonicBreakWindow = new SonicBreakApplication(event.timestamp)
+		this.SonicBreakApplications.push(sonicBreakWindow)
+	}
+
+	private breakUpdater(event: Event) : void {
+		const lastSonicBreak = this.lastSonicBreakApplication
+
+		//We missed the break maker somehow and now we need to make it.
+		if (lastSonicBreak == null || lastSonicBreak.end != null) {
+			this.breakMaker(event)
+		}
+
+		if (lastSonicBreak != null && lastSonicBreak.end == null) {
+			lastSonicBreak.totalTicks++
+		}
+	}
+
+	private breakEnder(event: Event) : void {
+		const lastSonicBreak = this.lastSonicBreakApplication
+
+		if (lastSonicBreak != null && lastSonicBreak.end == null) {
+
+			lastSonicBreak.end = event.timestamp
+
+			lastSonicBreak.isMissingTicks = lastSonicBreak.totalTicks < MAX_TICKS
+		}
+	}
+
+	private onComplete() {
+		const badSonicBreaks = this.SonicBreakApplications.filter(sonicBreak => sonicBreak.isMissingTicks)
+		if (badSonicBreaks.length > 0) {
+			const missedTicks = badSonicBreaks.reduce((acc, val) => acc + (MAX_TICKS - val.totalTicks), 0)
+			this.suggestions.add(new TieredSuggestion({
+				icon: this.data.actions.SONIC_BREAK.icon,
+				content: <Trans id="gnb.sonic-break.suggestions.missing-ticks.content">
+					One or more of your <DataLink action = "SONIC_BREAK"/> DoTs had ticks that did not deal damage. </Trans>,
+				tiers: {
+					1: SEVERITY.MINOR,
+					5: SEVERITY.MEDIUM,
+					10: SEVERITY.MAJOR,
+
+				},
+				value: missedTicks,
+				why: <Trans id = "gnb.sonic-break.suggestions.missing-ticks.why"> You wasted <Plural value= {missedTicks} one="# tick" other=" #ticks"/> of <DataLink action = "SONIC_BREAK"/> </Trans>,
```

Comment:
_Plural formatting:_ The space is in the wrong place for the `other` case, it should be `"# ticks"` instead of `" #ticks"`.